lib/matplotlib/axes/_axes.py

```diff
@@ -2011,6 +2011,26 @@ def step(self, x, y, *args, where='pre', data=None, **kwargs):
         kwargs['drawstyle'] = 'steps-' + where
         return self.plot(x, y, *args, data=data, **kwargs)
 
+    @staticmethod
+    def _convert_dx(dx, x0, x, convert):
+        """
+        Small helper to do logic of width conversion flexibly
+
+        *x0* and *dx* have units, but *x* has already been converted
```

Comment:
```suggestion
        *dx* and *x0* have units, but *x* has already been converted
```
(to match signature)